azurerm/internal/services/frontdoor/helper/helper.go

```diff
@@ -0,0 +1,85 @@
+package helper
```

Comment:
Could we just move this one up into the front door package?

Reply:
Moved.

Reply:
I tried moving it but go got confused with the path.